backend/functions/consolidations/consolidations.function.ts

```diff
@@ -25,24 +29,18 @@ export default async function handler(
     );
     const controller = new OrdersController(applicationContext);
     const procedure = applicationContext.request.params.procedure;
-    let response;
+    let response: ManageConsolidationResponse;
 
     if (procedure === 'reject') {
```

Comment:
When delegating handling the request to the controller from the function I would have had a single, consistently named function on each controller that would accept the CamsApplicationContext that would then have the routing logic in it, rather than having to inspect the request and contain logic in the Azure function.

Reply:
No objections from me. But not in this PR preferably.